jericho/jericho.py

```diff
@@ -522,8 +527,29 @@ def get_valid_actions(self, use_object_tree=True):
         interactive_objs  = self._identify_interactive_objects(use_object_tree=use_object_tree)
         best_obj_names    = self._score_object_names(interactive_objs)
         candidate_actions = self.act_gen.generate_actions(best_obj_names)
-        diff2acts         = self._filter_candidate_actions(candidate_actions)
-        valid_actions     = [max(v, key=utl.verb_usage_count) for v in diff2acts.values()]
+        if use_ctypes:
```

Comment:
I think this if statement would be better off within `_filter_candidate_actions`.